src/components/backend-ai-session-view.ts

```diff
@@ -431,15 +431,13 @@ export default class BackendAiSessionView extends BackendAIPage {
   render() {
     // language=HTML
     return html`
-      <div class="horizontal layout wrap">
```

Comment:
Why did you remove this line? If you remove this line, the resource statistics panel is too wide.

Reply:
This change is related to lablup/backend.ai-webui/pull/1117, lablup/backend.ai-webui/issues/1113.
I removed this line to make the width of resource statics panel and the width of session list panel equal.

Reply:
I got it. LGTM!